src/Parser/MistweaverMonk/CombatLogParser.js

```diff
@@ -0,0 +1,166 @@
+import getCastEfficiency from 'Parser/Core/getCastEfficiency';
+import ISSUE_IMPORTANCE from 'Parser/Core/ISSUE_IMPORTANCE';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+import DarkmoonDeckPromises from 'Parser/Core/Modules/Items/DarkmoonDeckPromises';
+import AmalgamsSeventhSpine from 'Parser/Core/Modules/Items/AmalgamsSeventhSpine';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+import Icon from 'common/Icon';
+import ITEMS from 'common/ITEMS';
+import ItemLink from 'common/ItemLink';
+import ItemIcon from 'common/ItemIcon';
+
+import StatisticBox from 'Main/StatisticBox';
+import SuggestionsTab from 'Main/SuggestionsTab';
+import TalentsTab from 'Main/TalentsTab';
+import CastEfficiencyTab from 'Main/CastEfficiencyTab';
+import CooldownsTab from 'Main/CooldownsTab';
+import ManaTab from 'Main/ManaTab';
+
+
+
+
+import CPM_ABILITIES, { SPELL_CATEGORY } from './CPM_ABILITIES';
+
+function formatThousands(number) {
+  return (Math.round(number || 0) + '').replace(/(\d)(?=(\d{3})+(?!\d))/g, "$1,");
+}
+function formatNumber(number) {
+  if (number > 1000000) {
+    return `${(number / 1000000).toFixed(2)}m`;
+  }
+  if (number > 10000) {
+    return `${Math.round(number / 1000)}k`;
+  }
+  return formatThousands(number);
+}
+function getIssueImportance(value, regular, major, higherIsWorse = false) {
+  if (higherIsWorse ? value > major : value < major) {
+    return ISSUE_IMPORTANCE.MAJOR;
+  }
+  if (higherIsWorse ? value > regular : value < regular) {
+    return ISSUE_IMPORTANCE.REGULAR;
+  }
+  return ISSUE_IMPORTANCE.MINOR;
+}
+function formatPercentage(percentage) {
+  return (Math.round((percentage || 0) * 10000) / 100).toFixed(2);
+}
+
+class CombatLogParser extends MainCombatLogParser {
+  static specModules = {
+    // Override the ability tracker so we also get stats for IoL and beacon healing
+    // abilityTracker: AbilityTracker,
+
+    // Features
+    // alwaysBeCasting: AlwaysBeCasting,
+    // cooldownTracker: CooldownTracker,
+
+
+    // Legendaries:
+    // drapeOfShame: DrapeOfShame,
+    // velens: Velens,
+    // prydaz: Prydaz,
+    // sephuz: Sephuz,
+
+
+    // Shared:
+    //amalgamsSeventhSpine: AmalgamsSeventhSpine,
+    //darkmoonDeckPromises: DarkmoonDeckPromises,
+  };
+
+  generateResults() {
+    const results = new ParseResults();
+
+
+    results.statistics = [
+      <StatisticBox
+        icon={(
+          <img
+            src="./img/healing.png"
+            style={{ border: 0 }}
+            alt="Healing"
+          />)}
+        value={`${formatNumber(this.totalHealing / this.fightDuration * 1000)} HPS`}
+        label={(
+          <dfn data-tip={`The total healing done recorded was ${formatThousands(this.totalHealing)}.`}>
+            Healing done
+          </dfn>
+        )}
+      />,
+    ];
+
+    results.items = [
+      /*this.modules.prydaz.active && {
```

Comment:
Prydaz and Sephuz should work out of the box (see DiscPriest). Velen's works if you configured the healing abilities array in the constants file.